projects/cashmere/src/lib/button/split-button/split-button.component.ts

```diff
@@ -83,10 +83,7 @@ export class SplitButtonComponent {
         this._tabIndex = value == null ? 0 : value;
     }
 
-    /**
-     * @deprecated
-     * @description Use `buttonStyle` instead
-     * */
+    /** **DEPRECATED** Use `buttonStyle` instead */
```

Comment:
When (if ever) do we plan to remove this deprecated property?  Also, curious why we aren't using the JSDoc comments for this anymore?

Reply:
Not sure we ever discussed a plan for pulling deprecated properties.  I suppose now would be as good a time as any - would that be your recommendation @joeskeen?  And the compiler was throwing an odd warning for the @deprecated tab, it also wasn't rendering very well in our template (just plain text).  So for all of the above I switched to what you see here.